src/Routes/index.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import Home from '../components/Home';
+import Counter from '../containers/Counter/Counter';
 
 function index() {
   return (
     <Switch>
+      <Route exact path="/" component={Counter} />
```

Comment:
I know it's an example to show it is working, but could you use different route pathname for them